### PR TITLE
feat(e2e): nv/pob integration testing p2 [ENG-1509]

### PR DESCRIPTION
**Pull request by nivasan1**
## In This PR 
- I migrate the second part of the POB v1 e2e tests
- The tests migrated over are the `MultiValidBundle` tests

### CODE REVIEW

File: tests/integration/chain_setup.go [hunk 1]
```diff
@@ -145,19 +146,24 @@ func BroadcastTxs(t *testing.T, ctx context.Context, chain *cosmos.CosmosChain,
...
+
+	for i, tx := range txs {
 		// broadcast tx
 		res, err := client.BroadcastTxSync(ctx, tx)
 		require.NoError(t, err)
+
 		// check execution was successful
-		require.Equal(t, res.Code, uint32(0))
+		if !msgsPerUser[i].ExpectFail {
+			require.Equal(t, res.Code, uint32(0))
```
Comment (davidterpay) [inline tests/integration/chain_setup.go:157]: this should be code 0?

File: tests/integration/pob_suite.go [hunk 2]
```diff
@@ -289,6 +296,363 @@ func (s *POBIntegrationTestSuite) TestValidBids() {
...
+		}
+		// create bid2 w/ higher bid than bid1
+		bid2, bundledTxs2 := CreateAuctionBidMsg(s.T(), context.Background(), s.user2, s.chain.(*cosmos.CosmosChain), bidAmt.Add(params.MinBidIncrement), []Tx{msg2})
+
+		// get chain height
+		height, err := s.chain.(*cosmos.CosmosChain).Height(context.Background())
+		require.NoError(s.T(), err)
+
+		// broadcast both bids
+		txs := BroadcastTxs(s.T(), context.Background(), s.chain.(*cosmos.CosmosChain), []Tx{
```
Comment (davidterpay) [inline tests/integration/pob_suite.go:342]: this broadcasting to the same validator

Comment (davidterpay) [reply tests/integration/pob_suite.go:342]: pls update the test case

File: tests/integration/pob_suite.go [hunk 3]
```diff
@@ -289,6 +296,363 @@ func (s *POBIntegrationTestSuite) TestValidBids() {
...
+		escrowAcctBalanceAfterBid := QueryAccountBalance(s.T(), s.chain, escrowAddr, params.ReserveFee.Denom)
+		expectedIncrement := escrowAddressIncrement(bidAmt.Add(params.MinBidIncrement.Add(bidAmt)).Amount, params.ProposerFee)
+		require.Equal(s.T(), escrowAcctBalanceBeforeBid+expectedIncrement, escrowAcctBalanceAfterBid)
+
+		// check next block
+		WaitForHeight(s.T(), s.chain.(*cosmos.CosmosChain), height+2)
+		block = Block(s.T(), s.chain.(*cosmos.CosmosChain), int64(height+2))
+
+		// check bid1 was included second
+		VerifyBlock(s.T(), block, 0, TxHash(txs[0]), bundledTxs)
```
Comment (davidterpay) [inline tests/integration/pob_suite.go:373]: check escrow balance here as well

File: tests/integration/pob_suite.go [hunk 4]
```diff
@@ -289,6 +296,363 @@ func (s *POBIntegrationTestSuite) TestValidBids() {
...
+		// check bid2 was included first
+		VerifyBlock(s.T(), block, 0, TxHash(txs[0]), bundledTxs)
+
+		// check escrow balance
+		escrowAcctBalanceAfterBid := QueryAccountBalance(s.T(), s.chain, escrowAddr, params.ReserveFee.Denom)
+		expectedIncrement := escrowAddressIncrement(bidAmt.Amount, params.ProposerFee)
+		require.Equal(s.T(), escrowAcctBalanceBeforeBid+expectedIncrement, escrowAcctBalanceAfterBid)
+	})
+
+	s.Run("Multiple bid transactions from diff accounts with second bid being smaller than min bid increment", func() {
```
Comment (davidterpay) [inline tests/integration/pob_suite.go:432]: this test case is essentially the same as the last no?

File: tests/integration/pob_suite.go [hunk 5]
```diff
@@ -289,6 +296,363 @@ func (s *POBIntegrationTestSuite) TestValidBids() {
...
+		txs := BroadcastTxs(s.T(), context.Background(), s.chain.(*cosmos.CosmosChain), []Tx{
+			{
+				User:   s.user1,
+				Msgs:   []sdk.Msg{bid2},
+				Height: height + 1,
+			},
+			{
+				User:              s.user1,
+				Msgs:              []sdk.Msg{bid1},
+				Height:            height + 1,
```
Comment (davidterpay) [inline tests/integration/pob_suite.go:521]: you have to increase the height here for the error to not be a timeout error

**Review by davidterpay — COMMENTED** [review]
looks like a good amount of these test cases do not match e2e currently. also noticed some duplication.

File: tests/integration/pob_suite.go (hunk 4 above)
Comment (nivasan1) [reply tests/integration/pob_suite.go:432]: Yea, I think so as well the difference is that bids are signed by diff accounts

File: tests/integration/pob_suite.go (hunk 5 above)
Comment (nivasan1) [reply tests/integration/pob_suite.go:521]: good point

File: tests/integration/pob_suite.go (hunk 2 above)
Comment (nivasan1) [reply tests/integration/pob_suite.go:342]: Discussed offline, functionality to be tested is that bids sent to same val can be committed over multiple blocks

**Review by davidterpay — APPROVED** [review]

File: tests/integration/chain_setup.go (hunk 1 above)
Comment (nivasan1) [reply tests/integration/chain_setup.go:182]: wym?

**Pull request merged**